### PR TITLE
Issue #11: Bundle geoPHP library

### PR DESCRIPTION
**Pull request by earthday47**

### CODE REVIEW

File: geofield.formatters.inc [hunk 1]
```diff
@@ -213,6 +213,8 @@ function geofield_field_formatter_settings_summary($field, $instance, $view_mode
  * Implements hook_field_formatter_view().
  */
 function geofield_field_formatter_view($entity_type, $entity, $field, $instance, $langcode, $items, $display) {
+  module_load_include('inc', 'geofield', 'vendor/phayes/geophp/geoPHP');
```
Comment (jenlampton) [inline geofield.formatters.inc]: we probably don't need `vendor/phayes/geophp` here. It's very unlikely that one module will ever include more than one library, or have more than one vendor.  Maybe replace with a single `libraries` directory instead, to match `css` and `js` like this: https://api.backdropcms.org/modules

edit: see comment on PR, I have a Q.

**Comment by jenlampton** [comment]
Thanks @earthday47! The changes to the module itself look quite simple, and very straightforward. :) 

> Issue #11: Add GeoPHP to project with composer.

Why with composer? Maybe I'm missing something, is there a licensing restriction? Ideally we'd copy/paste the whole library into `geofield/libraries` and be done.

**Comment by earthday47** [comment]
@jenlampton I just typed `composer require phayes/geoPHP` in the module root. That was the easiest for me to add the dependency. And an update would just be a `composer update`. But if that's not Backdrop Contrib's usual method, I can move it to `libraries` like you said.

**Comment by jenlampton** [comment]
It's fine if you want to use composer to manange the dependency locally, but maybe it would be better if we don't commit any of the composer stuff into the repo. Can you tell composer where to put the library? If so, maybe it can be both easy for you to maintain, and we can keep the repo clean. Maybe just put some of the local composer stuff into .gitignore files?

**Comment by earthday47** [comment]
@jenlampton making more sense to me, thanks for linking to that docs page. I updated to move it to `libraries/`. I'd still like to keep the cloned geophp folder as-is, tho, there's a ton of includes in that main geoPHP.inc one.

**Pull request merged**